### ISSUE TITLE
Unrecognised Type (Text)

### ISSUE THREAD

**Issue by travis-leith** [Giorgi/DuckDB.NET#15]
Using the latest version of the repo, and changing the sample code to the following

```c#
command.CommandText = "CREATE TABLE integers(foo text, bar INTEGER);";
var executeNonQuery = command.ExecuteNonQuery();

command.CommandText = "INSERT INTO integers VALUES ('3', 4), ('5', 6), ('7', NULL);";
```

and `PrintQueryResults` changed to

```c#
if (queryResult.GetFieldType(ordinal) == typeof(string))
                    {
                        var sval = queryResult.GetString(ordinal);
                        Console.Write(sval);
                    }
                    else
                    {
                        var val = queryResult.GetInt32(ordinal);
                        Console.Write(val);
                    }

```

I get the unrecognized type error.

But if I change the `DuckDBType` enum to 

```c#
DuckdbTypeVarchar = 17
```

then it works. This error existed on the previous version but the latest version of DuckDB.Net was released before I was finished investigating. Note that on the previous version the required value to get it work was 20, not 17. Does this mean that extra types should be inserted in the enum to get it up to at least 17?

**Comment by Giorgi** [comment]
The `DuckDBType` enum is missign some members.

**Comment by travis-leith** [comment]
How can I go about find which enum members are missing? Are they listed anywhere in the duckdb repo?

**Comment by Giorgi** [comment]
https://duckdb.org/docs/api/c/types

**Comment by Giorgi** [comment]
Fixed by 73171681cefc0a605ddd5895656b9e900d170381
